### PR TITLE
Tina MDX blog post

### PR DESCRIPTION
**Pull request by perkinsjr**
This will need to be merged on 12/09

### CODE REVIEW

File: content/blog/using-the-power-of-mdx-with-tina.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+``` bash,copy
+npx create-next-app -e with-tailwindcss tina-demo
+cd tina-demo
+```
+Step 2: Add Tina to the project
+
+Use the following command inside of the project to add all the Tina dependencies and wrap the application, ready to be used.
+
+``` bash,copy
+npx @tinacms/cli@latest init
```
Comment (jeffsee55) [inline content/blog/using-the-power-of-mdx-with-tina.md:23]: Should they choose to override the _app.js when this command asks them?

File: content/blog/using-the-power-of-mdx-with-tina.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+```
+Finally, we can update your `TinaMarkdown` component to pass the components for our users to use.
+
+```javascript,copy
+const components = {
+    Callout: (props) => {
+      return <Callout callout={props} />;
+    }};
+```
+
```
Comment (jeffsee55) [inline content/blog/using-the-power-of-mdx-with-tina.md:213]: This should be `<TinaMarkdown content={content} components={components} />`

**Review by jeffsee55 — COMMENTED** [review]
My only qualm with this is that we're showing them how to add this to an existing site (with `tinacms init`) but then we don't actually show them anything related to _their_ existing site's code since this is all done for the page we generated for them. So why not just use `npx create-tina-app` and use the barebones starter give them less things to think about?

File: content/blog/using-the-power-of-mdx-with-tina.md (hunk 2 above)
Comment (perkinsjr) [reply content/blog/using-the-power-of-mdx-with-tina.md:213]: Good catch! I had to redo the code because docs ended up breaking all the code off.

**Comment by perkinsjr** [comment]
@jeffsee55 
The reason we aren't doing tina-create-app is that I want to write about how we have now made it easier to create applications. This then allows us to go back and update this.

File: content/blog/using-the-power-of-mdx-with-tina.md (hunk 1 above)
Comment (perkinsjr) [reply content/blog/using-the-power-of-mdx-with-tina.md:23]: Yup. I should add that.

**Comment by jeffsee55** [comment]
Do you mean as a separate blog post? Would it make sense to drop that in before this one?

**Comment by perkinsjr** [comment]
> 

 Yup, a completely separate post. and no people tend not to read before December - January so this is under the radar. With that post going out early next year.

**Review by jeffsee55 — APPROVED** [review]

**Pull request merged**